### PR TITLE
feat(page): New About page

### LINKED ISSUE #326: Tentang Kami page
## UI Design

On Figma: https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=458%3A2632

## Implementation Model

Static Markdown file, editable from Netlify CMS, rendered as static page on Next.js.

## Tasks

- [ ] Initial content for Tentang Kami page with Netlify CMS
- [ ] Create design for Tentang Kami page
- [ ] Create markdown file + content widgets for Tentang Kami page on Netlify CMS
- [ ] Create page on frontend

### PR DESCRIPTION
**Pull request by mazipan**
Closes #326

## Description

Tentang kami Page

## Figma

https://www.figma.com/file/XNNtIoFEdFqaXOee83n0oN/WBW?node-id=485:2485

## Current Tasks

CMS on https://deploy-preview-616--wargabantuwarga.netlify.app/admin

- [x] Netlify CMS config for content body on #619
- [x] Netlify CMS config for list of videos on #617
- [x] Netlify CMS config for list of contributors on #618
- [x] Slicing Basic UI for About Page
- [x] Read the data from CMS: content markdown, videos and collaborators
- [x] Testing

### CODE REVIEW

**Comment by mazipan** [comment]
Who's have access to upload image to cloudinary ya @resir014 @zainfathoni?

**Comment by mazipan** [comment]
Need upload this:

About Page -- Main Image
gratis-sembako-ini-digantung-untuk-bantu-warga-terdampak-covid-19_169 1(1)

About Page -- Video Thumbnail 1

download (22) 1

About Page -- Video Thumbnail 2

download (21) 1

**Comment by mazipan** [comment]
For temporary, I will use my own Cloudinary

**Comment by resir014** [comment]
Okay, I'll upload them.

**Comment by resir014** [comment]
@mazipan 

About Page -- Main Image

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562337/128723750-85eeba33-e7fc-48c3-bcac-bec153de705b_ptasae.png

About Page -- Video Thumbnail 1

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562409/128723954-c3281899-8ef1-45cd-a942-48fcabb53b43_joubqm.png

About Page -- Video Thumbnail 2

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562441/128724036-e7a193ee-f27e-4439-a286-ede1b1260d29_suos7t.png

About Page -- Logo KitaBisa

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562465/128724320-3102fa65-3d99-4ba6-b045-e819ffe0364f_rc0scp.png

About Page -- Logo KawalCovid

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562487/128724490-c64239c5-ab17-498b-8267-314515b942f0_lziid2.png

About Page -- Logo FIM

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562551/128724600-f7c60a95-9ab9-4761-bcbb-a25fa3596a1a_npxy45.png

About Page -- Logo FZ

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562569/128724687-994afef6-944b-4e00-ad2d-752bd63c0c6b_krjmjo.png

About Page -- Logo Kata.ai (new Kata.ai logo)

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562739/kata-horizontal-blue_kjhwrf.png

About Page -- Logo IndoRelawan

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562784/128724925-ce9d18c2-6bd3-45ae-9311-c5bf0868bd82_thi3pq.png

About Page -- Logo Pemimpin.id

https://res.cloudinary.com/wargabantuwarga/image/upload/v1628562788/128725017-34897249-608b-4455-a24f-35468a6450bd_nqk3r8.png

File: pages/tentang-kami.tsx [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                    );
+                  })}
+                </div>
+              </div>
+            </div>
+
+            {/* Collaborators Sections --start */}
+            <div className="p-4 space-y-4 bg-white overflow-hidden shadow rounded-md">
+              <h2 className="text-center font-semibold text-gray-700 text-lg">
+                Terima kasih kepada para kolaborator inisiatif #WargaBantuWArga
```
Comment (resir014) [inline pages/tentang-kami.tsx:108]:
```suggestion
                Terima kasih kepada para kolaborator inisiatif #WargaBantuWarga
```

**Review by resir014 — CHANGES_REQUESTED** [review]

**Review by resir014 — APPROVED** [review]

**Comment by mazipan** [comment]
Wah, belum ta ubah pake Next/Imagec :-D

**Pull request merged**